manifests/params.pp

```diff
@@ -62,6 +68,14 @@
   $email_from = "no-reply@${::domain}"
   $email_enabled = false
 
+  $lazy_enabled = undef
+  $lazy_redirect_host = undef
+  $lazy_redirect_port = undef
+  $lazy_redirect_path = undef
+  $lazy_https_retrieval = undef
+  $lazy_download_interval = undef
+  $lazy_download_concurrency = undef
```

Comment:
I think all of these should have default value, whever the pulp server.conf default is